WriteHeader/calibration.py

```diff
@@ -0,0 +1,23 @@
+import os
+from numpy import matrix as M
+from calibration_tool import CalibrationTool
+
+basedir = 'C:\Users\chris\Desktop\Calibration\CadaverData'
```

Comment:
For little utility script, you typically use something like [argparse](https://docs.python.org/3/library/argparse.html) to set the arguments through the command line. This is fine for now though.